pkg/gui/services/custom_commands/handler_creator.go

```diff
@@ -272,19 +275,26 @@ func (self *HandlerCreator) finalHandler(customCommand config.CustomCommand, ses
 			cmdObj.StreamOutput()
 		}
 		output, err := cmdObj.RunWithOutput()
+
+		if refreshErr := self.c.Refresh(types.RefreshOptions{Mode: types.ASYNC}); err != nil {
```

Comment:
Switching to an async refresh because it's faster and we don't need to wait for it to complete before doing some other action